### ISSUE TITLE
Same symbol pattern each time?

### ISSUE THREAD

**Issue by Yellowcrayon5x5** [maxcleme/beadifier#81]
I was curious if with the perler midis if the patterns use the same symbols for every picture? For example, will p18 (black) always translate into a capital L? Or does it depend upon the pattern and color palate itself?

**Comment by maxcleme** [comment]
Hi,

The symbols generation has been designed to stay the same, whatever colors
you have enabled or disabled. As far as I remember, it should also comes
with a 'exponent' letter if multiple palettes are used across the same
pattern.

Are you experiencing undesired behaviors?

Regards.

**Comment by Yellowcrayon5x5** [comment]
No I am not having any issues. I just happen to be neurotic and wanted to make a piece of paper to have as a guide. I intended on using this site alot and I am not familiar with the numbers of the beads. I wanted to make a reference guide if you will. I have also noticed that it has gingerbread listed as number 259 but the perler bags are listed as 250

Sent from Yahoo Mail on Android 
 
  On Sat, Jun 19, 2021 at 3:39 PM, ***@***.***> wrote:   

Hi,

The symbols generation has been designed to stay the same, whatever colors
you have enabled or disabled. As far as I remember, it should also comes
with a 'exponent' letter if multiple palettes are used across the same
pattern.

Are you experiencing undesired behaviors?

Regards.
